main/models.py

```diff
@@ -25,8 +25,8 @@ def __str__(self):
         return self.title
 
     def approved_comments(self):
-        """Approving comments. Use for moderating."""
-        return self.comments.filter(approved_comment=True)
+        """Will show comments that are ok in your opinion."""
```

Comment:
`Will show` is a Future Tense. You need Present Simple `Show`.
Not `your opinion`, but `someone's opinion`.